rs/nns/constants/src/lib.rs

```diff
@@ -149,7 +157,14 @@ pub const ALL_NNS_CANISTER_IDS: [&CanisterId; 10] = [
     &GENESIS_TOKEN_CANISTER_ID,
     &IDENTITY_CANISTER_ID,
     &NNS_UI_CANISTER_ID,
+    &ICP_LEDGER_ARCHIVE_CANISTER_ID,
```

Comment:
it's a bit confusing to me to have zero based indexing but then not call this `ICP_LEDGER_ARCHIVE_0_CANISTER_ID`. isn't that inconsistent?